frontend/app/routes/public/renew/$id/adult-child/review-adult-information.tsx

```diff
@@ -374,21 +375,47 @@ export default function RenewAdultChildReviewAdultInformation() {
               </DescriptionListItem>
             </dl>
           </section>
+          {!hasChildren && (
+            <section className="space-y-4">
+              <h2 className="font-lato text-2xl font-bold">{t('renew-adult-child:review-child-information.submit-app-title')}</h2>
+              <p className="mb-4">{t('renew-adult-child:review-child-information.submit-p-proceed')}</p>
+              <p className="mb-4">{t('renew-adult-child:review-child-information.submit-p-false-info')}</p>
+              <p className="mb-4">{t('renew-adult-child:review-child-information.submit-p-repayment')}</p>
+            </section>
+          )}
         </div>
         <fetcher.Form onSubmit={handleSubmit} method="post" className="mt-8 flex flex-row-reverse flex-wrap items-center justify-end gap-3">
           <input type="hidden" name="_csrf" value={csrfToken} />
-          {hCaptchaEnabled && <HCaptcha size="invisible" sitekey={siteKey} ref={captchaRef} />}
-          <LoadingButton
-            id="confirm-button"
-            name="_action"
-            value={FormAction.Submit}
-            variant="green"
-            disabled={isSubmitting}
-            loading={isSubmitting && submitAction === FormAction.Submit}
-            data-gc-analytics-customclick="ESDC-EDSC:CDCP Renew Application Form-Adult:Submit - Review your information click"
-          >
-            {t('renew-adult-child:review-adult-information.submit-button')}
-          </LoadingButton>
+          {!hasChildren && hCaptchaEnabled && <HCaptcha size="invisible" sitekey={siteKey} ref={captchaRef} />}
```

Comment:
Nitpick: Grouping the related logic together makes things a bit more cleaner IMO:
```jsx
{hasChildren && (
  <LoadingButton /> 
)}
{!hasChildren && (
  <>
    {hCaptchaEnabled && <HCaptcha />
    <LoadingButton /> 
  <>
)}
```